cmd/registry/cmd/apply/testdata/sample/apis/registry.yaml

```diff
@@ -25,7 +25,7 @@ data:
         state: Staging
         specs:
           - metadata:
-              name: openapi.yaml
+              name: openapi
```

Comment:
Since we export resources into YAML files, I think we want to avoid creating resources with names that include ".yaml" extensions. This was an arbitrary convention introduced by the OpenAPI bulk uploader that I think we can easily change.